backends/advanced/scripts/sync_friendlite_mycelia.py

```
@@ -0,0 +1,382 @@
+
+
+class FriendLiteMyceliaSync:
+    """Sync Friend-Lite users with Mycelia OAuth credentials."""
+
+    def __init__(self, mongo_url: str, mycelia_db: str, friendlite_db: str):
+        self.mongo_url = mongo_url
+        self.mycelia_db = mycelia_db
+        self.friendlite_db = friendlite_db
+        self.client = MongoClient(mongo_url)
+
+        print(f"📊 Connected to MongoDB:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)